baybe/utils/random.py

```diff
@@ -2,10 +2,17 @@
 
 import contextlib
 import random
+import warnings
 
 import numpy as np
+from typing_extensions import deprecated
 
 
+@deprecated(
+    "Using 'set_random_seed' is deprecated and support will be removed in a future "
+    "release. Use the new settings management system instead. For details: "
+    "https://emdgroup.github.io/baybe/stable/userguide/settings.html)",
```

Comment:
is that bracket intended / needed?
```suggestion
    "https://emdgroup.github.io/baybe/stable/userguide/settings.html",
```